development/webpack/test/plugins.ManifestPlugin.test.ts

```diff
@@ -281,4 +281,62 @@ describe('ManifestPlugin', () => {
       }
     }
   });
+
+  describe('manifest flags in development mode', () => {
+    const testManifest = {} as chrome.runtime.Manifest;
+    const mockFlags = { remoteFeatureFlags: { testFlag: true } };
+
+    it('adds manifest flags in development mode', () => {
+      const transform = transformManifest(
+        { lockdown: true, test: false },
+        true,
+      );
+      assert(transform, 'transform should be truthy');
+
+      // Mock fs.readFileSync
+      const fs = require('fs');
+      const originalReadFileSync = fs.readFileSync;
+      fs.readFileSync = () => JSON.stringify(mockFlags);
```

Comment:
NIT:

Node's testing library has mocking built it. I think the right way to do this is to create a `before` block within this new `'manifest flags in development mode'` `describe` block:

```typescript
    // (note: this is untested)
    const originalReadFileSync = fs.readFileSync;
    before("mock `.manifest-flags.json`", () => {
      mock.method(fs, 'readFileSync', (path: string, options: object) => {
        // only override reads for `.manifest-flags.json`:
        if (path === resolve(__dirname, <the path to the file here>)) {
          // mock `.manifest-flags.json`
          return JSON.stringify(mockFlags);
        }
        return originalReadFileSync(path, options);
      });
    });
    
    after(() => mock.restoreAll());
```

Reply:
Since there are 2 different return response, i didn't add the mock to beforeEach, and keep it within each test addressed af74af6c36cd3313fd0008bd3d82611d9cc63014

Reply:
Mocks look great. You do still need to restore the originals after each test though.

Adding

```typescript
afterEach(() => mock.restoreAll());
```

within the `describe` callback that contains these tests (`describe('manifest flags in development mode', () => {`) is probably the easiest/best way to do it.

note: I had previously suggested using `after`, but that is not correct; it should be `afterEach`.